### PR TITLE
Set element to 'X' for unknown/ambiguous cases.

### PR DESCRIPTION
**Pull request by JoaoRodrigues**
This change comes from my benchmark on the entire PDB. I added a small test to assign elements and we have a few failures on unknown or ambiguous atoms. Here's an example from PDB 4CPA:

```
ATOM   2439  N   GLX I   2     -14.463 -16.664  -1.443  1.00 23.73           N  
ATOM   2440  CA  GLX I   2     -13.386 -16.604  -2.426  1.00 29.49           C  
ATOM   2441  C   GLX I   2     -12.828 -15.227  -2.806  1.00 31.92           C  
ATOM   2442  O   GLX I   2     -11.632 -15.318  -3.276  1.00 37.34           O  
ATOM   2443  CB  GLX I   2     -13.815 -17.229  -3.786  1.00 23.88           C  
ATOM   2444  CG  GLX I   2     -13.280 -16.825  -5.123  1.00 25.96           C  
ATOM   2445  CD  GLX I   2     -12.527 -17.669  -6.090  1.00 35.96           C  
ATOM   2446  XE1 GLX I   2     -11.768 -18.704  -5.944  1.00 40.06           X  
ATOM   2447  XE2 GLX I   2     -12.724 -17.305  -7.323  1.00 35.50           X  
```

Currently, our parsers assign an empty/blank element to those `XE1` and `XE2` atoms (as they should, good thing they don't assign xenon!) but it seems that the PDB prefers these are 'X'. This PR makes that change and moves some of the documentation from function comments to the docstring.

Not urgent to include this, happy to wait until we release 1.78.

- [x] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [x] I have read the ``CONTRIBUTING.rst`` file, have run ``pre-commit`` locally,
and understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [x] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

**Comment by JoaoRodrigues** [comment]
@jgreener64 - would appreciate a small review here if you have the time!

**Comment by jgreener64** [comment]
Would this mean that reading in a PDB file with no elements listed and then writing it back out would put Xs in all the element entries?

Not that that's such a bad thing, I'm just wondering.

**Comment by JoaoRodrigues** [comment]
Only for those atoms for which we could not successfully assign an element
to. Others would have regular C, N, etc elements.

**Comment by jgreener64** [comment]
Ah that makes sense. This PR looks good to me.

**Comment by JoaoRodrigues** [comment]
Thanks!

File: NEWS.rst [hunk 1]
```diff
@@ -8,12 +8,20 @@ https://www.open-bio.org/category/obf-projects/biopython/
 
 The latest news is at the top of this file.
 
-(In progress, not yet released): Biopython 1.78
```
Comment (peterjc) [inline NEWS.rst:11]: Whoops. Good catch.

**Review by peterjc — APPROVED** [review]
Style wise this looks fine, and I understand the rational. No comment on the wisdom of this change, I never had to deal with atypical atoms in PDB files.

**Comment by JoaoRodrigues** [comment]
There isn't much wisdom here, just trying to match the PDB 'standard' as much as possible and this is one of those 'undocumented' features... Thanks for the comments! Will merge!

**Pull request merged**